pvlib/test/test_epw.py

```diff
@@ -0,0 +1,22 @@
+import inspect
+import os
+
+from pandas.util.testing import network
+
+from pvlib.iotools import epw
+
+test_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+epw_testfile = os.path.join(test_dir, '../data/NLD_Amsterdam062400_IWEC.epw')
```

Comment:
`epw_testfile = os.path.join(test_dir, 'data', 'NLD_Amsterdam062400_IWEC.epw')`

Reply:
All data files for other functions are in `pvlib/data` instead of `pvlib/test/data`.
Maybe this needs to be restructured at a general level, but for this PR I think it's better to stay with the common approach.

Reply:
the preferred pattern is demonstrated here: https://github.com/pvlib/pvlib-python/blob/master/pvlib/test/test_solrad.py#L11-L14

Reply:
Done. A nicer solution indeed.

Reply:
yeah, sorry I didn't catch that earlier. we need to update the older files in another PR.